client/src/components/Board/Column.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Card, CardActions, CardContent, CardHeader } from '@material-ui/core';
+import { Droppable, Draggable } from 'react-beautiful-dnd';
+import useStyles from './useStyles';
+import Task from './Task';
+import { Column as ColumnInterface } from '../../interface/Column';
+import { Task as TaskInterface, TaskPlaceHolder } from '../../interface/Task';
+import { useState } from 'react';
+export interface properties {
+  placeHolderStyle: TaskPlaceHolder;
+  Column: ColumnInterface;
+  index: number;
+  Tasks: TaskInterface[];
+}
+
+const Column: React.FunctionComponent<properties> = (props) => {
+  const [state, setState] = useState({ placeHolderStyle: props.placeHolderStyle, visible: -1, dragFinished: true });
+  const classes = useStyles();
+  const setVisable = (visible: number) => {
+    setState({ ...state, visible });
+  };
+  const onMouseEnter = () => {
+    const placeHolder = document.getElementById('placeholder-' + props.Column.Id);
+    const column = document.getElementById(props.Column.Id);
+    if (!placeHolder || !column) {
+      return;
+    }
+    if (
+      column.getBoundingClientRect().bottom <
+      placeHolder.getBoundingClientRect().bottom + placeHolder.getBoundingClientRect().height * 0.5
+    ) {
+      setVisable(-1);
+    } else {
+      setVisable(1);
+    }
+  };
+  const onMouseLeave = () => {
+    setVisable(-1);
+  };
+  const onMouseUp = () => {
+    setVisable(-1);
+  };
+
+  return (
+    <Draggable draggableId={props.Column.Id} index={props.index}>
+      {(provided) => (
+        <Card
+          className={classes.column}
+          {...provided.draggableProps}
+          ref={provided.innerRef}
```

Comment:
Should be able to add a disableElevation property onto this which would allow the columns to have no drop shadow which follows mock-ups more.